Gems/Atom/RPI/Code/Source/RPI.Reflect/Model/ModelLodAssetCreator.cpp

```diff
@@ -53,14 +53,14 @@ namespace AZ
             }
         }
 
-        void ModelLodAssetCreator::SetMeshAabb(AZ::Aabb&& aabb)
+        void ModelLodAssetCreator::SetMeshAabb(AZ::Aabb aabb)
```

Comment:
The compiler might optimize this already, but ideally it should be `const AZ::Aabb&` to avoid the double copy.

Reply:
Will fix